ndarray/src/main/java/org/tensorflow/ndarray/impl/sparse/hydrator/DoubleSparseNdArrayHydrator.java

```diff
@@ -0,0 +1,126 @@
+package org.tensorflow.ndarray.impl.sparse.hydrator;
+
+import org.tensorflow.ndarray.DoubleNdArray;
+import org.tensorflow.ndarray.NdArrays;
+import org.tensorflow.ndarray.Shape;
+import org.tensorflow.ndarray.hydrator.DoubleNdArrayHydrator;
+import org.tensorflow.ndarray.hydrator.NdArrayHydrator;
+import org.tensorflow.ndarray.impl.sparse.DoubleSparseNdArray;
+
+public class DoubleSparseNdArrayHydrator implements DoubleNdArrayHydrator {
```

Comment:
Javadoc?

Reply:
In general, there are no need of Javadoc for classes under `impl`